Engine/UI/ComponentsUI.h

```diff
@@ -0,0 +1,38 @@
+#ifndef _COMPONENTSUI_H_
```

Comment:
As anabel said this file shouldn't exist.

Reply:
I cannot find in the code that line

Reply:
Deleted